server/stream/streamBroadcast.coffee

```diff
@@ -63,6 +61,82 @@ authorizeConnection = (instance) ->
 				connections[instance].disconnect()
 				delete connections[instance]
 
+	Meteor.methods
+		broadcastAuth: (remoteId, selfId) ->
+			check selfId, String
+			check remoteId, String
+
+			@unblock()
+			if selfId is InstanceStatus.id() and remoteId isnt InstanceStatus.id() and InstanceStatus.getCollection().findOne({_id: remoteId})?
+				@connection.broadcastAuth = true
+
+			return @connection.broadcastAuth is true
+
+		stream: (streamName, eventName, args) ->
+			# Prevent call from self and client
+			if not @connection?
+				return 'self-not-authorized'
+
+			# Prevent call from unauthrorized connections
+			if @connection.broadcastAuth isnt true
+				return 'not-authorized'
+
+			if not Meteor.StreamerCentral.instances[streamName]?
+				return 'stream-not-exists'
+
+			Meteor.StreamerCentral.instances[streamName]._emit(eventName, args)
+
+			return undefined
+
+startCentralBroadcast = (value) ->
+	instance = 'BROADCAST_CENTRAL'
+
+	logger.connection.info 'connecting in', instance, value
+	connection = DDP.connect(value, {_dontPrintErrors: true})
+	connections[instance] = connection
+	connection.instanceId = instance
+	connection.onReconnect = ->
+		authorizeConnection(instance)
+
+	connection._stream.on 'message', (raw_msg) ->
+		msg = DDPCommon.parseDDP(raw_msg)
+		if not msg or msg.msg isnt 'changed' or not msg.collection? or not msg.fields?
+			return
+
+		{streamName, eventName, args} = msg.fields
+
+		if not streamName? or not eventName? or not args?
+			return
+
+		console.log 'on message', msg.fields
+
+		if connection.broadcastAuth isnt true
+			return 'not-authorized'
+
+		if not Meteor.StreamerCentral.instances[streamName]?
+			return 'stream-not-exists'
+
+		Meteor.StreamerCentral.instances[streamName]._emit(eventName, args)
+
+	connection.subscribe 'stream'
+
+@connections = {}
+@startStreamBroadcast = () ->
+	process.env.INSTANCE_IP ?= 'localhost'
```

Comment:
@continuouslee Yes, but that will not stop RC, you will need to refresh the page to see new messages.